DellPEWSManTools/Public/PEDRAC/Find-PEDRAC.ps1

```diff
@@ -174,7 +174,7 @@ Function Find-PEDRAC
                             }
                             catch 
                             {
-                                Write-Error 
+                                Write-Verbose "No iDRAC found at $ip"
```

Comment:
@micmaher Good catch!
The verbose message is good enough but we should surface the error thrown by adding the below.

```powershell
Write-Error -ErrorRecord $PSitem
```